README.md

```diff
@@ -22,6 +22,57 @@ Feel free to inquire about its usage by creating an issue in this repository.
 To find syntax for search queries, check out the [documentation on searching issues and pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests)
 or the [documentation on searching discussions](https://docs.github.com/en/search-github/searching-on-github/searching-discussions).
 
+## Quick Start
```

Comment:
```suggestion
## Getting Started
```

Reply:
I like your suggestion from the PR description of using the term "Getting Started".